### DISCUSSION TITLE
Table 2

### DISCUSSION THREAD

**Discussion by maja-nagler** [IBAHCM/RPiR#531] [Registration]
At our table are: @rayewb @sas-b9 @cheisbaee @maja-nagler 
This is my link: https://github.com/SBOHVM/maja-naglerSeries01.git

**Comment by sas-b9** [comment]
MINE IS (SASKIA BAKER) https://github.com/SBOHVM/githubsas-b9Series01.git

**Reply by maja-nagler** [reply]
Everything works :)

**Reply by cheisbaee** [reply]
All good here :)

**Comment by rayewb** [comment]
Raye's - https://github.com/SBOHVM/rayewbSeries01.git

**Reply by maja-nagler** [reply]
Awesome :) Works

**Reply by sas-b9** [reply]
Lookin fab!

**Reply by cheisbaee** [reply]
Great job, it works :)

**Comment by cheisbaee** [comment]
https://github.com/SBOHVM/cheisbaeeSeries01.git

**Reply by maja-nagler** [reply]
Looks great to me.

**Reply by sas-b9** [reply]
Looks fab!

**Comment by maja-nagler** [comment]
This is my link (open for comments): https://github.com/SBOHVM/maja-naglerSeries01.git

**Reply by cheisbaee** [reply]
Looks great :) Great job :)

**Reply by sas-b9** [reply]
Looks fab!

**Comment by G-Shiach** [comment]
Hello, Sorry this is wildly late, I have finally figured what the discussion thing meant (in hindsight it should've been simpler).
This is my link https://github.com/SBOHVM/G-ShiachSeries01.git
I hope that works, sorry again

**Comment by cheisbaee** [comment]
Hi everyone, this is the link to my series 02 repository:

https://github.com/SBOHVM/cheisbaeeSeries02.git

Let me know if any issues arise when running my code. @sas-b9 @maja-nagler @rayewb 

Che

**Comment by maja-nagler** [comment]
Hi,
This is the link to my second repository:
https://github.com/SBOHVM/maja-naglerSeries02.git

Please let me know if You spot any mistakes :)

**Comment by rayewb** [comment]
Hi,

This is the link to my second repository:

https://github.com/SBOHVM/rayewbSeries02.git
